README.md

```diff
@@ -109,6 +109,38 @@ const catWatch = new ServiceClient({
 });
 ```
 
+## Retry Logic
+
+For application critical requests it can be a good idea to retry failed requests to the responsible services.
+
+Occasionaly target server can have high latency for a short period of time, or in the case of a stack of servers, one server can be having issues
+and retrying the request will allow perron to attempt to access one of the other servers that currently aren't facing issues.
+
+By default `perron` has retry logic implemented, but configured to perform 0 retries. Internally `perron` [node-retry](https://github.com/tim-kos/node-retry) to handle the retry logic
+and configuration. All of the existing options provided by `node-retry` can be passed via configuration options through `perron`.
+
+There is also the addition of a transient error check function. This can be defined in any way by the consumer and is used in the try logic to determine whether to 
+attempt the retries or not depending on the type of error. If the function returns true and the number of retries hasn't been exceeded, the request can be retried.
+
+```js
+const ServiceClient = require('perron');
+
+const catWatch = new ServiceClient({
+    hostname: 'catwatch.opensource.zalan.do',
+    // These are the default settings
+    retryOptions: {
```

Comment:
I think that README should provide useful example options, since that is what users will copy-paste by default. If I read [node-retry documentation](https://github.com/tim-kos/node-retry#retrytimeoutsoptions) formula right, current config will result in a not linear distribution between 200 and 300ms with more than half delays being 300ms.

Reply:
Any recommendations as to what you would prefer here? Currently this example is the same as what the default retryOptions are configured in perron, so I should also update there if you believe that the current config is not appropriate

Reply:
I would just change `maxTimeout` to 400 probably

Reply:
Ok no worries, I'll update :)